software-engineering-policies/TechnicalDebt/TechnicalDebtPolicy.md

```diff
@@ -1,48 +1,172 @@
 # Technical Debt Policy
 
+All UKHO employees engaged with digital product development should develop a basic understanding of tehcnical debt, how it's generated, it's impacts on the business and how it can be paid off.
```

Comment:
```suggestion
All UKHO employees engaged with digital product development should develop a basic understanding of technical debt, how it's generated, its impacts on the business and how it can be paid off.
```